.github/workflows/needinfo-stale.yml

```diff
@@ -0,0 +1,22 @@
+---
+name: Close old issues with the needinfo label
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Close old issues with the needinfo tag
+        uses: dwieeb/needs-reply@v2
```

Comment:
Let's explicitly set `days-before-close` so we don't have to check the action's documentation to figure out when an issue is considered old.

Reply:
Done, thanks!